### PR TITLE
[Fix] typo in some docker image labels `k9s` instead of `k8s`

### PR DESCRIPTION
**Pull request by jstourac**
This is a very simple typo fix into the docker image label. I believe we don't need any extra testing.

This typo caused that there were these four labels in the respective docker images:

```
io.k8s.description
io.k8s.display-name
io.k9s.description
io.k9s.display-name
```

Related to #310

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by jstourac** [comment]
@harshad16 do you want me to create an issue for this?

**Comment by harshad16** [comment]
@jstourac yes please, if you can create an issue
that would be great.

**Comment by jstourac** [comment]
Created #310.

File: base/c9s-python-3.9/Dockerfile [hunk 1]
```diff
@@ -3,8 +3,8 @@ FROM quay.io/sclorg/python-39-c9s:c9s
 LABEL name="odh-notebook-base-centos-stream9-python-3.9" \
       summary="Python 3.9 CentOS Stream 9 base image for ODH notebooks" \
       description="Base Python 3.9 builder image based on CentOS Stream 9 for ODH notebooks" \
-      io.k9s.display-name="Python 3.9 c9s base image for ODH notebooks" \
-      io.k9s.description="Base Python 3.9 builder image based on C9S for ODH notebooks" \
+      io.k8s.display-name="Python 3.9 c9s base image for ODH notebooks" \
```
Comment (atheo89) [inline base/c9s-python-3.9/Dockerfile:6]: lgtm

**Review by atheo89 — APPROVED** [review]
/lgtm

**Pull request merged**